client/rpc-servers/src/lib.rs

```diff
@@ -38,7 +41,10 @@ pub const WS_MAX_BUFFER_CAPACITY_DEFAULT: usize = 16 * MEGABYTE;
 /// Default maximum number of connections for WS RPC servers.
 const WS_MAX_CONNECTIONS: usize = 100;
 
-/*/// RPC server-specific prometheus metrics.
+pub mod middleware;
+
+// TODO: (dp) Wire up the ServerMetrics as well
```

Comment:
ServerMetrics is a way to get the number of open connections assuming that it only applies to WebSocket.

Reply:
that's is annoying to support it would need additional refactoring in jsonrpsee....